tiledb/bioimg/converters/axes.py

```diff
@@ -221,11 +221,11 @@ def canonical(self, shape: Tuple[int, ...]) -> Axes:
         dims = frozenset(dim for dim, size in zip(self.dims, shape) if size > 1)
         return Axes(dim for dim in self.CANONICAL_DIMS if dim in dims)
 
-    def mapper(self, other: Axes) -> AxesMapper:
+    def mapper(self, other: Axes) -> CompositeAxesMapper:
```

Comment:
Is this necessary? Is the only subclass of `AxesMapper` that should be returned by the function?